docs/development/dev_howto.rst

```diff
@@ -668,6 +668,21 @@ If you have a alternative attribute to use instead, pass its name in the `altern
     print(some_class(10).old_attribute)
 
 
+How to capture a deprecation
+++++++++++++++++++++++++++++
+
+It is important to also catch the deprecation warning in the test files. This can be done in the following way:
```

Comment:
```suggestion
A deprecation warning raised during CI will be considered an error by `pytest`. Nevertheless, it is important to also check that deprecation warnings are correctly emitted in the test files. This can be done in the following way:
```